builtin/logical/rabbitmq/path_config_lease.go

```diff
@@ -0,0 +1,103 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"lease": &framework.FieldSchema{
```

Comment:
Please use `ttl` and `ttl_max` -- since about 0.3 we've been using these to avoid confusion with the concept of the `lease` itself.